### DISCUSSION TITLE
EEIO 1.1.1 vs 1.3

### DISCUSSION THREAD

**Discussion by Wab1l** [USEPA/supply-chain-factors#83] [Q&A] [answered]
Which of these data sets are more recommended to use as the 1.1.1 data set has more options to chose from, whereas the 1.3 version being the latest is a lot different from it, can i use the 1.1.1 by using the inflation.

**Comment by bl-young** [comment, marked as answer]
We always recommend using the latest version of the factors as they reflect the most up to date data and modeling. For example, the GHG data in v1.3 are from those published earlier this year, while v1.1.1 is several years older. I'm not sure what you mean by there are more options to choose from in v1.1.1? They file structure may have changed since then, but both v1.2 and v1.3 use a NAICS based sector classification structure which means they have more unique sector codes available.

**Reply by Wab1l** [reply]
I mean that in the v1.1.1, there are summary and detail sheets for commodities and industries, meanwhile in v1.3, its only industries, so for now i have been using v1.1.1 and i used the cotton emission factor cause i purchase cotton from a textile manufacturing, however due the change versions now there is Cotton Ginning Factor Available as well which is the industry from which i purchase cotton, previously that cotton ginning was named Agriculture and Forestry Support, can you guide me on this?, Thanks @bl-young

**Reply by bl-young** [reply]
Because v1.2 and later used NAICS categorization, the sector name labels are more indicative of an industry designation, however the underlying model is still a [commodity model](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.2.22-GHG.yml#L10).

You are correct that in more recent versions we are not publishing the summary level datasets anymore as these are quite aggregated. So the aggregated "Agriculture and Forest Support" commodity no longer is available as a supply chain factor. You would need to choose one of the factors within that grouping.
